benchmarks/htap/lib/controller.py

```diff
@@ -54,57 +67,72 @@ def oltp_worker(self, worker_id):
         # do NOT introduce timeouts for the tpcc queries! this will make that
         # the workload gets inbalanaced and eventually the whole benchmark stalls
         with DBConn(self.args.dsn) as conn:
-            tpcc_worker = Transactions(worker_id, self.scale_factor, self.latest_timestamp, conn, self.args.dry_run)
+            tpcc_worker = Transactions(
+                worker_id,
+                self.scale_factor,
+                self.latest_timestamp,
+                conn,
+                self.args.dry_run,
+            )
             next_reporting_time = time.time() + 0.1
             while True:
                 self.oltp_sleep()
                 tpcc_worker.next_transaction()
                 if next_reporting_time <= time.time():
                     # its beneficial to send in chunks so try to batch the stats by accumulating 0.1s of samples
-                    self.stats_queue.put(('tpcc', tpcc_worker.stats()))
+                    self.stats_queue.put(("tpcc", tpcc_worker.stats()))
                     next_reporting_time += 0.1
 
-
     def olap_worker(self, worker_id):
-        queries = Queries(worker_id, self.args, self.range_delivery_date[0],
-                          self.latest_timestamp, self.stats_queue)
+        queries = Queries(
+            worker_id,
+            self.args,
+            self.range_delivery_date[0],
+            self.latest_timestamp,
+            self.stats_queue,
+        )
         while True:
             queries.run_next_query()
 
     def _sql_error(self, msg):
         import sys
-        print(f'ERROR: {msg} Did you run `prepare_benchmark`?')
+
```

Comment:
Weird new line here

Reply:
I guess it's because of the `import` statement. I couldn't find mention of such a rule in the code style, though.

https://black.readthedocs.io/en/stable/the_black_code_style.html

Reply:
@0xfeedface could you maybe check that Black does it with PEP-8 compliancy in mind?

Reply:
According to the black website, it is PEP-8 compliant: https://github.com/psf/black#using-black-with-other-tools